openvdb/openvdb/tree/RootNode.h

```diff
@@ -1143,6 +1176,9 @@ struct RootNodeCopyHelper<RootT, OtherRootT, /*Compatible=*/true>
         };
 
         self.mBackground = Local::convertValue(other.mBackground);
+#if OPENVDB_ABI_VERSION_NUMBER >= 10
+        self.mOrigin = other.mOrigin;
```

Comment:
This seems odd to me? Shouldn't the copy constructor and the assignment operator have identical behavior in that they both set the origin to zero because this is all that is supported in this version (and ideally warn or error if the incoming grid is non-zero) ?